src/Runner.Worker/JobRunner.cs

```diff
@@ -49,6 +49,8 @@ public async Task<TaskResult> RunAsync(AgentJobRequestMessage message, Cancellat
                 VssCredentials jobServerCredential = VssUtil.GetVssCredential(systemConnection);
                 await runServer.ConnectAsync(systemConnection.Url, jobServerCredential);
                 server = runServer;
+
+                _jobServerQueue.Start(message, enableActionsUpload: false);
```

Comment:
`_jobServerQueue`  isn't initialized here because it is only initialized in the else block